uberfire-m2repo-editor/uberfire-m2repo-editor-backend/src/main/java/org/guvnor/m2repo/backend/server/helpers/HttpGetHelper.java

```diff
@@ -75,6 +75,11 @@ public void handle(final HttpServletRequest request,
         final File file = new File(mavenRootDir,
                                    requestedFile);
 
+        if(file == null){
```

Comment:
Please format properly (missing space after if and after `)`)

Reply:
Fixed